doc/shared/Install.rst

```diff
@@ -1034,6 +1046,22 @@ illustration only.
 
    Default: None
 
+.. cmakeoption:: ENABLE_ADIAK
+   
+   Enable Adiak support
+
+   Default: OFF
+
+   .. note::
+      
+      Using Adiak requires setting :cmakeop:`ENABLE_CALIPER` and :cmakeop:`SUNDIALS_BUILD_WITH_PROFILING` to ``ON``.
```

Comment:
Does it still require Caliper? Or did you update it so that Adiak can be turned on without Caliper?

Reply:
Adiak's been updated to be able to be on without Caliper.